### DISCUSSION TITLE
Is there any dashboard in Grafana that uses Prometheus metrics for Authelia?

### DISCUSSION THREAD

**Discussion by webysther** [authelia/authelia#6376] [Q&A] [answered]
Don't find any here (https://grafana.com/grafana/dashboards/) and no more than the docs about enabling the metrics

**Comment by james-d-elliott** [comment, marked as answer]
We do not formally have any dashboards for it, but would welcome a contribution under the Apache 2.0 license and can upload them under our org once they are made.

**Reply by webysther** [reply]
I made one simple to getting started, if you point me a place to create the json or docs I will open a PR

**Reply by webysther** [reply]
![image](https://github.com/authelia/authelia/assets/750007/547dc0f2-a7cd-4de0-957e-a767ec90f4d0)

**Reply by james-d-elliott** [reply]
Maybe `examples/grafana-dashboards/<description>.json`?

**Comment by stefangweichinger** [comment]
would like to test that dashboard as well. thanks @webysther for your work

**Reply by stefangweichinger** [reply]
ah, found it, thanks
